Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,14 +1,75 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.IO;
+using static System.Math;
 using Console = System.Console;
 
 namespace BootCamp.Chapter
 {
     class Program
     {
+        /*  Homework 3
+            1) Fork https://github.com/csinn/CSharp-From-Zero-To-Hero/tree/Chapter1/Homework/3 
+            2) Take homework 2 code, place it in the forked branch and refactor it using functions. There should be as little duplicate code as possible (there should be functions for: 
+                - Calculating BMI, 
+                - Prompt for input and converting it to int, 
+                - Prompt for input and converting it to string, 
+                - Prompt for input and converting it to float. 
+            3) Put all the function you made into the right places in Checks
+            4) Move all the code from main function into lesson3 class function called Demo. Call it from main function.
+        */
+
         static void Main(string[] args)
         {
+            Lesson3.Demo();
+        }
+        public static string AddNewPerson()
```

Comment:
Leave program only with the main method. All other methods have a better place- which is the Lesson class.

Reply:
Done